Blogpost about UFRN Nextflow graduate course (#127)
nextflow-io/website: content/blog/2023/nextflow-goes-to-university.md

title=Nextflow goes to university!

> date=2023-07-34

Reviewer: ```suggestion
date=2023-07-24
```

Author: Oooops thanks 😅

type=post

> description=Helpful tips for troubleshooting out-of-memory errors in containerized execution environments.

Reviewer: Needs updating

Author: This I missed indeed 🤣

> description=Seqera partnership with universities allows students to learn Nextflow and get credits for that!

Reviewer: ```suggestion
description=Seqera partnership with universities to help students to learn Nextflow
```

> image=img/to-be-decided.jpg

Reviewer: Needs this 😊

Author: Yup, I left this for last haha
